Automattic/jetpack
# Stats: improve i18n for plural forms in the Stats Widget

Fixes #20005

#### Changes proposed in this Pull Request:

The primary change is the introduction of the `_n` function to add the option to include singular and plural versions of the ‘Views’ word in the Stats Widget, when viewed in the `wp-admin` dashboard. This is also relevant for translations.
 
#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

On a Jetpack connected live testing site, on the main dashboard (at `wp-admin/index.php`) view the Stats Widget on the dashboard when you have a post with 1 view and another with multiple views. What you should see is ‘x Views’ for multiple views, and ‘1 View’ for a singular view (when viewing in English).

<img width="429" alt="Stats dashboard with singular and plural options" src="https://user-images.githubusercontent.com/16754605/126701533-99535e8f-ec2c-48e0-bc46-cc28718103f2.png">
